atlas/pixi.toml

```
@@ -15,14 +15,14 @@ description = "Get started with the repository"
 depends-on = ["lab"]
 
 [dependencies]
-notebook = ">=7.4.5,<8"
-jupyterlab = ">=4.4.5,<5"
-coffea = ">=2025.7.3,<2026"
-mplhep = ">=0.4.0,<0.5"
-xrootd = ">=5.8.1,<6"
+notebook = ">=7.4.7,<8"
+jupyterlab = ">=4.4.9,<5"
+mplhep = ">=0.4.1,<0.5"
+xrootd = ">=5.8.4,<6"
 fsspec-xrootd = ">=0.5.1,<0.6"
 ipywidgets = ">=8.1.7,<9"
+coffea = ">=2025.10.1,<2026"
+atlas-schema = ">=0.4.1,<0.5"
 
 [pypi-dependencies]
-atlas-schema = { git = "https://github.com/scipp-atlas/atlas-schema.git", rev = "feat/systematicsHandling" }
-atlasopenmagic = { git = "https://github.com/atlas-outreach-data-tools/atlasopenmagic", rev = "main" }
+atlasopenmagic = ">=1.6.1, <2"
```

```suggestion
atlas-schema = ">=0.4.1,<0.5"
atlasopenmagic = ">=1.6.1, <2"
```

`atlasopenmagic` is also on conda-forge now: https://github.com/conda-forge/atlasopenmagic-feedstock